src/main/java/co/inventorsoft/academy/service/FileWriterService.java

```diff
@@ -1,20 +1,22 @@
 package co.inventorsoft.academy.service;
 
 import lombok.AllArgsConstructor;
+import lombok.NoArgsConstructor;
 import org.springframework.stereotype.Service;
 
 import java.io.BufferedWriter;
 import java.io.IOException;
 @Service
-@AllArgsConstructor
+@NoArgsConstructor
 public class FileWriterService {
     /**
      * This method write to file String value of str parameter
      * @param str String value
      */
     public void write(String str) {
         try {
-            BufferedWriter writer = new BufferedWriter(new java.io.FileWriter("data.csv", true));
+            BufferedWriter writer = new BufferedWriter(
+                    new java.io.FileWriter("data.csv", true));
```

Comment:
You can write it in one line. Also, if you call your write method many times it would be better to create FileWriter only once as a class field.